air/src/preparation_step/preparation.rs

```diff
@@ -152,6 +159,13 @@ fn make_exec_ctx(
     Ok(ctx)
 }
 
+fn unjs_call_results(call_results: CallResults) -> Result<HashMap<u32, CallServiceResult>, ParseIntError> {
+    call_results
+        .into_iter()
+        .map(|(id, res)| id.parse().map(|id| (id, res)))
```

Comment:
It is hard to read this line b/c both variable names are the same in both map() calls.

Reply:
That's same value represented differently.  I think it is OK for Rust.